doorstop/cli/utilities.py

```diff
@@ -112,8 +112,11 @@ def configure_settings(args):
     # Parse `publish` settings
     if hasattr(args, 'no_child_links') and args.no_child_links is not None:
         settings.PUBLISH_CHILD_LINKS = args.no_child_links is False
-    if hasattr(args, 'no_body_levels') and args.no_body_levels is not None:
-        settings.PUBLISH_BODY_LEVELS = args.no_body_levels is False
+    if hasattr(args, 'no_body_levels'):
+        settings.PUBLISH_BODY_LEVELS = not args.no_body_levels
```

Comment:
Please add a deprecation warning here so this option can be safely removed in the 2.0 release:

```python
msg = "'--no-body-levels' option will be removed in a future release"
warnings.warn(msg, DeprecationWarning)
```